server/spec/unit/controllers/Cards/deletion.hook.js

```diff
@@ -0,0 +1,9 @@
+import { before } from 'hooks'
```

Comment:
Guess file should be under `system` domain, not `unit` 😉

Reply:
Anyway that's not needed, because we already have backend for that

BTW, we need to prepare testing database before several steps, we can do it via PP if you want (separate infra task can be created for that)

Reply:
@AleksSenkou ok, let's do it tomorrow

Reply:
@AleksSenkou opened task for that https://trello.com/c/R2QnJBZa/45-when-i-develop-app-id-like-to-write-different-sorts-of-test